dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1372,15 +1372,24 @@ def vander(x1, N=None, increasing=False):
     return call_origin(numpy.vander, x1, N=N, increasing=increasing)
 
 
-def zeros(shape, dtype=None, order='C'):
+def zeros(shape,
+          *,
+          dtype=numpy.float64,
+          order="C",
+          like=None,
+          device=None,
+          usm_type="device",
```

Comment:
```suggestion
          usm_type=None,
```
to align with dpnp.asarray() and dpnp.full().
Or is there a reason why dpctl.tensor has different default value for `usm_type` (even in zeros_like() it is `None`) ? Or probably a support of `None` is just missing in dpctl and needs to be added?

Reply:
For `asarray`, the `usm_type=None` is take to understand as infer the `usm_type` of the result from input array (if present). This does not apply to `zeros`. Default value of `usm_type` should be `"device"`.